usr/share/rear/build/default/990_verify_rootfs.sh

```diff
@@ -5,6 +5,31 @@
 
 LogPrint "Testing that the recovery system in $ROOTFS_DIR contains a usable system"
 
+if test "$KEEP_BUILD_DIR" = "errors"; then
+    local keep_build_dir_on_errors=1
+else
+    # KEEP_BUILD_DIR does not say to keep it on errors
+    # - effective value depends on whether we are running interactively
+    if tty -s ; then
+        local keep_build_dir_on_errors=1
+    else
+        local keep_build_dir_on_errors=0
+    fi
+fi
+
+function keep_build_dir() {
+    if ! is_true "$KEEP_BUILD_DIR" && ! is_false "$KEEP_BUILD_DIR"; then
+        # is either empty or equal to "errors" ... or some garbage value
+        KEEP_BUILD_DIR="${keep_build_dir_on_errors}"
+    fi
+    if is_true "$KEEP_BUILD_DIR" ; then
+        LogPrintError "Build area kept for investigation in $BUILD_DIR, remove it when not needed"
```

Comment:
@pcahyna 
from plain looking at the code I think there are now
(i.e. since KEEP_BUILD_DIR has ternary semantics)
unneeded LogPrintError messages because ther is no need
to tell the user what happens when he has explicitly set
KEEP_BUILD_DIR to true or fales or "errors".
I think the only case where ReaR needs to tell the user
is the default case when KEEP_BUILD_DIR=""
where ReaR does its automated way of setting it.
Then the user should be informed what ReaR actually did.

Reply:
If it is set to true or "errors", the message is still helpful, as it contains the location of the build directory (which is otherwise not shown without using -v).
I agree in the case of setting it to false. This is what my last commit fixes.